src/uwtools/tests/api/test_ungrib.py

```diff
@@ -0,0 +1,34 @@
+# pylint: disable=missing-function-docstring
+
+import datetime as dt
+from unittest.mock import patch
+
+from uwtools.api import ungrib
+
+
+def test_execute(tmp_path):
+    dot = tmp_path / "graph.dot"
+    args: dict = {
+        "config_file": "config.yaml",
+        "cycle": dt.datetime.utcnow(),
+        "batch": False,
+        "dry_run": True,
+        "graph_file": dot,
+    }
+    with patch.object(ungrib, "Ungrib") as Ungrib:
```

Comment:
```suggestion
    with patch.object(ungrib, "_Ungrib") as Ungrib:
```
You'll need this when you change `Ungrib` to `_Ungrib` to mark it private in the `uwtools.api.ungrib` imports.